client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -215,30 +287,34 @@ const CancelPurchaseRefundInformation = ( {
 				} else {
 					text.push(
 						i18n.translate(
-							'This plan includes the custom domain, %(domain)s, normally a %(domainCost)s purchase. ' +
-								'The domain will not be removed along with the plan, to avoid any interruptions for your visitors. ',
+							'This plan subscription includes the custom domain, %(domain)s' +
+								'{{strong}}The domain will not be removed{{/strong}} along with the plan, to avoid any interruptions for your visitors. ',
```

Comment:
Is this missing a period between the two sentences?

Reply:
I added a period and <br /> as in the previous case